lib/ansible/plugins/callback/default.py

```diff
@@ -54,6 +54,15 @@ def __init__(self):
         self._last_task_name = None
         super(CallbackModule, self).__init__()
 
+    def get_option(key, default=None):
+        # use 'non standard' get_option as other plugins derive from this one and might not document options correctly
+        option = default
+        try:
+            option = super(CallbackModule, self).get_option(key)
+        except (AttributeError, KeyError):
+            pass
+        return option
+
```

Comment:
Why not just add this logic to the base class's `get_option()` function?

Reply:
because you should not need a default, this is a special case for 3rd party derived modules from default

Reply:
Sure, but this assumes that there is nowhere else that this kind of thing comes up. I've heard of people doing their own private connection plugin that inherits from `ssh` and modifies some behaviors. With as many options that are specified by the config fragment in the `ssh` connection plugin's docs, this same situation could easily crop up.

Reply:
we already handle that, but mostly on try/except instead